### PR TITLE
fix: change descriptions and password rule placement

### PR DESCRIPTION
**Pull request by hifabienne**

### CODE REVIEW

File: internal/api/ui/login/static/i18n/it.yaml [hunk 1]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
   SecretLabel: Chiave
   CodeLabel: Codice
   NextButtonText: Avanti
   CancelButtonText: annulla
 
 InitMFAU2F:
-  Title: Configurazione a più fattori U2F / WebAuthN
-  Description: Aggiungi il tuo Token fornendo un nome e cliccando sul pulsante 'Registra'.
-  TokenNameLabel: Nome del token / dispositivo
+  Title: Impostazione di U2F a 2 fattori / WebAuthN
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:98]:
```suggestion
  Title: Imposta U2F a 2 fattori / WebAuthN
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 2]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
+  Description: Aggiungete il vostro dispositivo dipendente da 2 fattori come FaceID, Windows Hello o Impronte digitali fornendo un nome (ad es. MyPhone, MacBook) e facendo clic sul pulsante "Registra 2 fattori" qui sotto.
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:99]:
```suggestion
  Description: Aggiungi il tuo dispositivo da 2 fattori come FaceID, Windows Hello o Impronte digitali, fornendo un nome (ad es. MyPhone, MacBook) e facendo clic sul pulsante "Registra" qui sotto.
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 3]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
+  TokenNameLabel: Nome del dispositivo a 2 fattori
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:100]:
```suggestion
  TokenNameLabel: Nome del dispositivo a due fattori
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 4]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
   NotSupported: WebAuthN non è supportato dal tuo browser. Assicurati che sia aggiornato o usane uno diverso (ad esempio Chrome, Safari, Firefox)
-  RegisterTokenButtonText: Registra
+  RegisterTokenButtonText: Registrazione a 2 fattori
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:102]:
```suggestion
  RegisterTokenButtonText: Registra
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 5]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
   ErrorRetry: Riprova, crea una nuova richiesta o scegli un metodo diverso.
 
 InitMFADone:
-  Title: Verificazione a più fattori effettuata
-  Description: La verificazione del multifattore eseguita con successo. Il multifattore è richiesto ad ogni login.
+  Title: Verifica a 2 fattori effettuata
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:106]:
```suggestion
  Title: Verificazione a due fattori effettuata
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 6]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
+  Description: La verifica a 2 fattori è stata eseguita con successo. Il secondo fattore deve essere inserito a ogni accesso.
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:107]:
```suggestion
  Description: La verificazione a due fattori è stata eseguita con successo. Il secondo fattore deve essere inserito a ogni accesso.
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 7]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
-  Provider0: OTP (One Time Password)
-  Provider1: U2F (2° fattore universale)
+  Provider0: App Autenticatore (ad esempio Google/Microsoft Authenticator, Authy)
+  Provider1: Dipende dal dispositivo (ad es. FaceID, Windows Hello, impronta digitale)
   ChooseOther: o scegli un'altra opzione
 
 VerifyMFAOTP:
-  Title: Verificazione del Multificator
-  Description: Verifica il tuo multifattore
+  Title: Verifica a 2 fattori
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:117]:
```suggestion
  Title: Verificazione fattore
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 8]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
+  Description: Verificare il secondo fattore
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:118]:
```suggestion
  Description: Verifica il tuo secondo fattore con la tua app
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 9]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
   CodeLabel: Codice
   NextButtonText: Avanti
 
 VerifyMFAU2F:
-  Title: Verificazione a più fattori
-  Description: Verifica il tuo token U2F / WebAuthN
+  Title: Verifica a 2 fattori
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:123]:
```suggestion
  Title: Verificazione fattore
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 10]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
+  Description: Verificare il fattore 2 con il dispositivo registrato (ad es. FaceID, Windows Hello, impronta digitale).
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:124]:
```suggestion
  Description: Verifica il tuo fattore con il dispositivo registrato (ad es. FaceID, Windows Hello, impronta digitale).
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 11]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
   NotSupported: WebAuthN non è supportato dal tuo browser. Assicurati di avere l'ultima versione installata o usane una diversa (per esempio Chrome, Safari, Firefox).
   ErrorRetry: Prova di nuovo, crea una nuova richiesta o scegli un metodo diverso.
-  ValidateTokenButtonText: Verifica
+  ValidateTokenButtonText: Verifica a 2 fattori
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:127]:
```suggestion
  ValidateTokenButtonText: Verifica
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 12]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
 
 Passwordless:
   Title: Accesso senza password
-  Description: Verifica il tuo token
+  Description: Accedere con i metodi di autenticazione forniti dal dispositivo, come FaceID, Windows Hello o impronta digitale.
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:131]:
```suggestion
  Description: Accedi con il metodo di autenticazione del tuo dispositivo registrato (ad es. FaceID, Windows Hello o impronta digitale).
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 13]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
   NotSupported: WebAuthN non è supportato dal tuo browser. Assicurati che sia aggiornato o usane uno diverso (ad esempio Chrome, Safari, Firefox)
   ErrorRetry: Riprova, crea una nuova richiesta o scegli un metodo diverso.
   LoginWithPwButtonText: Accedi con password
-  ValidateTokenButtonText: Verifica
+  ValidateTokenButtonText: Accedi senza password
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:135]:
```suggestion
  ValidateTokenButtonText: Accedi
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 14]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
 
 PasswordlessPrompt:
   Title: Autenticazione passwordless
-  Description: Vuoi impostare il login senza password?
+  Description: Vuoi impostare il login senza password? Metodi di autenticazione del dispositivo, come FaceID, Windows Hello o impronte digitali.
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:139]:
```suggestion
  Description: Vuoi impostare il login senza password? Scegli tra metodi di autenticazione di un dispositivo (ad es. FaceID, Windows Hello o impronte digitali).
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 15]
```diff
@@ -181,7 +181,7 @@ EmailVerification:
   Description: Ti abbiamo inviato un'e-mail per verificare il tuo indirizzo. Inserisci il codice nel campo sottostante.
   CodeLabel: Codice
   NextButtonText: Avanti
-  ResendButtonText: rispedisci
+  ResendButtonText: codice di reinvio
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:184]:
```suggestion
  ResendButtonText: Reinvia
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 16]
```diff
@@ -172,7 +172,7 @@ PasswordChangeDone:
   NextButtonText: Avanti
 
 PasswordResetDone:
-  Title: Link per il cambiamento inviato
+  Title: Invio del link per la reimpostazione della password
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:175]:
```suggestion
  Title: Link per la reimpostazione della password è stato inviato
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 17]
```diff
@@ -172,7 +172,7 @@ PasswordChangeDone:
...
   Description: Controlla la tua email per reimpostare la tua password.
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:176]:
```suggestion
  Description: Controlla la tua email per continuare e reimpostare la tua password.
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 18]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
+  TokenNameLabel: Nome del dispositivo
   NotSupported: WebAuthN non è supportato dal tuo browser. Assicurati che sia aggiornato o usane uno diverso (ad esempio Chrome, Safari, Firefox)
-  RegisterTokenButtonText: Registra
+  RegisterTokenButtonText: Registrazione senza password
   ErrorRetry: Riprova, crea una nuova richiesta o scegli un metodo diverso.
 
 PasswordlessRegistrationDone:
   Title: Configurazione dell'autenticazione senza password
-  Description: Token per lautenticazione passwordless aggiunto con successo.
+  Description: Dispositivo per passwordless aggiunto con successo.
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:155]:
```suggestion
  Description: Dispositivo per l'autenticazione passwordless aggiunto con successo.
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 19]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
 
 PasswordlessRegistration:
   Title: Configurazione dell'autenticazione senza password
-  Description: Aggiungi il tuo Token fornendo un nome e poi cliccando sul pulsante 'Registra'.
-  TokenNameLabel: Nome del token / dispositivo
+  Description: Add your authentication by providing a name (e.g MyMobilePhone, MacBook, etc) and then clicking on the 'Register passwordless' button below.
+  TokenNameLabel: Nome del dispositivo
   NotSupported: WebAuthN non è supportato dal tuo browser. Assicurati che sia aggiornato o usane uno diverso (ad esempio Chrome, Safari, Firefox)
-  RegisterTokenButtonText: Registra
+  RegisterTokenButtonText: Registrazione senza password
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:150]:
```suggestion
  RegisterTokenButtonText: Registra
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 20]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
   Title: Autenticazione passwordless
-  Description: Vuoi impostare il login senza password?
+  Description: Vuoi impostare il login senza password? Metodi di autenticazione del dispositivo, come FaceID, Windows Hello o impronte digitali.
   DescriptionInit: Devi impostare il login senza password. Usa il link che ti è stato inviato per registrare il tuo dispositivo.
   PasswordlessButtonText: Continua
   NextButtonText: Avanti
   SkipButtonText: salta
 
 PasswordlessRegistration:
   Title: Configurazione dell'autenticazione senza password
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:146]:
```suggestion
  Title: Registrazione dell'autenticazione passwordless
```

File: internal/api/ui/login/static/i18n/it.yaml [hunk 21]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
-  Description: Aggiungi il tuo Token fornendo un nome e poi cliccando sul pulsante 'Registra'.
-  TokenNameLabel: Nome del token / dispositivo
+  Description: Add your authentication by providing a name (e.g MyMobilePhone, MacBook, etc) and then clicking on the 'Register passwordless' button below.
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:147]:
```suggestion
  Description: Aggiungi il tuo metodo fornendo un nome (ad es. Cellulare, MacBook, etc) e poi cliccando sul pulsante 'Registra'.
```

**Review by peintnermax — CHANGES_REQUESTED** [review]

File: internal/api/ui/login/static/i18n/it.yaml (hunk 15 above)
Comment (hifabienne) [reply internal/api/ui/login/static/i18n/it.yaml:184]: its about mention the code in the button. that was somethin we found out in the user tests

File: internal/api/ui/login/static/i18n/it.yaml [hunk 22]
```diff
@@ -64,12 +64,12 @@ InitPasswordDone:
...
 InitUser:
   Title: Attivare l'utente
-  Description: Hai ricevuto un codice, che devi inserire nel modulo sottostante, per verificare la tua email e impostare la tua nuova password.
+  Description: Verifica la tua e-mail con il codice sottostante e imposta la tua nuova password.
   CodeLabel: Codice
   NewPasswordLabel: Nuova password
   NewPasswordConfirm: Conferma la password
   NextButtonText: Avanti
-  ResendButtonText: rispedisci
+  ResendButtonText: reinvio del codice
```
Comment (hifabienne) [inline internal/api/ui/login/static/i18n/it.yaml:72]:
```suggestion
  ResendButtonText: Reinvia codice
```

**Review by livio-a — CHANGES_REQUESTED** [review]
there's a layout issue
image

File: internal/api/ui/login/static/i18n/zh.yaml [hunk 23]
```diff
@@ -78,81 +78,81 @@ InitUserDone:
...
   CodeLabel: 验证码
   NextButtonText: 继续
   CancelButtonText: 取消
 
 InitMFAU2F:
-  Title: 多因素身份认证设置 U2F / WebAuthN
-  Description: 通过提供名称添加您的令牌，然后单击下面的“注册令牌”按钮。
-  TokenNameLabel: 令牌/机器的名称
+  Title: 设置2-Factor U2F / WebAuthN
+  Description: 通过提供一个名称（如MyPhone，MacBook），然后点击下面的 "注册2-factor "按钮，添加你的设备依赖的2-factor，如FaceID，Windows Hello或指纹。
```
Comment (jinrenjie) [inline internal/api/ui/login/static/i18n/zh.yaml:99]: fix typo: 
`通过提供一个名称（如MyPhone，MacBook），然后点击下面的 "注册2-Factor" 按钮，添加支持 2-Factor （如FaceID，Windows Hello或指纹）的设备。`

**Review by jinrenjie — COMMENTED** [review]
Translation check completed.

**Comment by jinrenjie** [comment]
> there's a layout issue image

Consider using some localization management tool to manage language packs? If only the text is translated, the translator does not know the context in which the text is located, resulting in unsatisfactory translation results and unpredictable UI rendering effects.

**Comment by peintnermax** [comment]
> there's a layout issue image

fixed 👍

File: internal/api/ui/login/static/i18n/it.yaml [hunk 24]
```diff
@@ -104,7 +104,7 @@ InitMFAU2F:
 
 InitMFADone:
   Title: Verificazione a due fattori effettuata
-  Description: La verificazione a due fattori è stata eseguita con successo. Il secondo fattore deve essere inserito a ogni accesso.
+  Description: Fantastico! Avete appena impostato con successo il vostro fattore 2 e reso il vostro account molto più sicuro. Il secondo fattore deve essere inserito a ogni accesso.
```
Comment (peintnermax) [inline internal/api/ui/login/static/i18n/it.yaml:107]:
```suggestion
  Description: Fantastico! Hai appena impostato un secondo fattore e quindi reso il tuo account molto più sicuro. Il secondo fattore deve essere inserito a ogni accesso.
```

**Review by peintnermax — CHANGES_REQUESTED** [review]

**Review by peintnermax — APPROVED** [review]

**Pull request merged**